src/main/java/fi/aalto/cs/apluscourses/ui/module/ModuleListView.java

```diff
@@ -1,29 +1,26 @@
 package fi.aalto.cs.apluscourses.ui.module;
 
+import com.intellij.ui.ListSpeedSearch;
 import fi.aalto.cs.apluscourses.presentation.module.ModuleListElementViewModel;
-import fi.aalto.cs.apluscourses.ui.ComponentUtil;
 import fi.aalto.cs.apluscourses.ui.base.BaseListView;
 
-import java.awt.font.TextAttribute;
-
 import org.jetbrains.annotations.NotNull;
 
-
 public class ModuleListView
-    extends BaseListView<ModuleListElementViewModel, ModuleListElementView> {
-
-
+        extends BaseListView<ModuleListElementViewModel, ModuleListElementView> {
+  /**
+   * Constructs a view for the module list.
+   */
   public ModuleListView() {
     super(new ModuleListElementView());
+    setFixedCellHeight(26);
```

Comment:
![Screenshot_128](https://user-images.githubusercontent.com/73069541/106666465-4d45c680-65b0-11eb-9989-bdaf8f5d5825.png)

Reply:
This is how it looks without that `setFixedCellHeight`, and it seems like changing the font size does not influence the spacing (i.e. the spacing is not preserved, which is not good)

Reply:
It doesn't have to look "good" with a non-default font size but ideally (for better accessibility), the text should remain visible even if font size were increased.

I wonder if it's possible to use fontsize- bound length units like in HTML/CSS. If it was, that could perhaps be a better  choice there.

It doesn't have to be fixed here, though. This code is ok, as-is.

Reply:
This might be possible by calling `setFixedCellHeight` with a parameter dependant on `list.getFont().getSize()` - however, you'd have to somehow catch the event of the font being changed (unlikely that something like this exists, however I haven't checked), set the cell height AND refresh the list

Reply:
There's no reason to go that far for an issue this small, so let's then just keep it as it is in this PR. :+1: